install_themes/action.yml

```diff
@@ -54,3 +54,55 @@ runs:
             ${{ inputs.container }} \
             composer update ${{ inputs.update_options }}
         fi
+
+    - name: Generate deprecation warning to summary page
+      shell: bash
+      run: |
+        echo "Deprecation Warning:" >> $GITHUB_STEP_SUMMARY
+          echo "---------------------" >> $GITHUB_STEP_SUMMARY
+          echo -e "\033[0;31m### This actions are deprecated and may not be supported ${{ inputs.install_theme}} \033[0m" >> $GITHUB_STEP_SUMMARY
+          exit 1
+
+    - name: 'Output report'
+      shell: bash
+      run: |
+        for FILE in $(find reports -type f|sort); do
+          cat "${FILE}" >>github_report.md
+        done
+        cat github_report.md >>"${GITHUB_STEP_SUMMARY}"
+        
+        for FILE in $(find slack_report -type f|sort); do
```

Comment:
Why are you looping over a non-existent folder and consolidate non-existent files into a variable which is then not used? (line 74 to 77)